bundles/org.eclipse.text/src/org/eclipse/jface/text/AbstractDocument.java

```diff
@@ -242,15 +243,6 @@ protected List<IDocumentPartitioningListener> getDocumentPartitioningListeners()
 		return asList(fDocumentPartitioningListeners);
 	}
 
-	/**
-	 * Returns all positions managed by the document grouped by category.
-	 *
-	 * @return the document's positions
-	 */
-	protected Map<String, List<Position>> getDocumentManagedPositions() {
```

Comment:
I don't understand why API tooling doesn't complain here, but removing protected method from public class seems like API breakage for me.

Reply:
Actually API tooling complained after explicit project build.